airframe-http-recorder/src/main/scala/wvlet/airframe/http/recorder/HttpRecorder.scala

```diff
@@ -53,8 +53,14 @@ case class HttpRecorderConfig(destUri: String,
   */
 object HttpRecorder extends LogSupport {
 
-  // Ignore Finagle's tracing IDs.
-  def defaultExcludeHeaderPrefixes: Seq[String] = Seq("date", "x-b3-", "finagle-")
+  // Http headers to ignore for recording and hashing purposes
+  def defaultExcludeHeaderPrefixes: Seq[String] = Seq(
```

Comment:
By removing unstable HTTP headers by default, we don't need to extend HttpRecorderServer to make it programmable.